### PR TITLE
LPS-190132 [EVP] Update Request Status When Organization Is Review

### PR DESCRIPTION
**Pull request by Elvinho**

### CODE REVIEW

**Comment by Elvinho** [comment]
ci:test:stable

File: workspaces/liferay-evp-workspace/settings.gradle [hunk 1]
```diff
@@ -0,0 +1,24 @@
+buildscript {
+	dependencies {
+		classpath group: "biz.aQute.bnd", name: "biz.aQute.bnd.gradle", version: "5.2.0"
+		classpath(group: "com.liferay", name: "com.liferay.gradle.plugins.workspace", version: "6.1.13") {
```
Comment (nNilton) [inline workspaces/liferay-evp-workspace/settings.gradle:4]: please check the plugin version

File: workspaces/liferay-evp-workspace/gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.9.2-bin.zip
```
Comment (nNilton) [inline workspaces/liferay-evp-workspace/gradle/wrapper/gradle-wrapper.properties:3]: check version

File: workspaces/liferay-evp-workspace/gradle.properties [hunk 3]
```diff
@@ -0,0 +1,12 @@
+##
+## Check GETTING_STARTED.markdown for additional properties and their default
+## values.
+##
+
+liferay.workspace.bundle.dist.include.metadata=true
+liferay.workspace.modules.dir=modules
+liferay.workspace.node.package.manager=yarn
+liferay.workspace.product=dxp-7.4-u81
```
Comment (nNilton) [inline workspaces/liferay-evp-workspace/gradle.properties:9]: check version

**Review by nNilton — CHANGES_REQUESTED** [review]

**Comment by Elvinho** [comment]
ci:test:sf

**Comment by Elvinho** [comment]
ci:test:sf

**Comment by Elvinho** [comment]
ci:test:sf

**Comment by Elvinho** [comment]
ci:test:sf

**Comment by Elvinho** [comment]
ci:test:relevant

**Comment by Elvinho** [comment]
ci:forward

**Pull request closed without merging**